.github/workflows/csharp-integration.yml

```diff
@@ -0,0 +1,33 @@
+name: C# Generator Integration Test
```

Comment:
The `#` will be interpreted as a comment.

Reply:
haha, I can see that in the github code block too. I should have known

thanks!